src/components/shared/Search/Search.tsx

```diff
@@ -17,21 +25,28 @@ export default function Search({ labelBouton, placeholder }: Props): ReactElemen
         className="fr-input"
         id="search-784-input"
         name="search-784-input"
+        onChange={rechercher}
         placeholder={placeholder}
         type="search"
+        value={termesDeRecherche}
       />
       <button
         className="fr-btn"
+        onClick={soumettreLaRecherche}
         title={labelBouton}
         type="button"
```

Comment:
```suggestion
        type="submit"
```
Un seul _hook_ (pas de `soumettreLaRechercheEnter`).